.github/workflows/post-release.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Set env
         run: |
           echo "LIFECYCLE_VERSION=$(echo ${{ github.event.release.tag_name }} | cut -d "v" -f2)" >> $GITHUB_ENV
-          echo "LIFECYCLE_IMAGE_TAG=$(git describe --always --dirty)" >> $GITHUB_ENV
+          echo "LIFECYCLE_IMAGE_TAG=$(git describe --always)" >> $GITHUB_ENV
```

Comment:
Removing `--dirty` makes sense - was it there to guard producing images for a repo with uncommitted code?

Reply:
Way way back when we first decided to auto build & tag lifecycle images, there was the concept of `--dirty` because we thought that would be useful in local development. But it turned out to be more confusing than useful, and we dropped it. I don't think we ever want to publish `<some-sha>-dirty` tagged images to buildpacksio, though it could happen go.sum changes are uncommitted, hence this removal.